frcxx_nft/src/util.rs

```diff
@@ -0,0 +1,97 @@
+use fvm_shared::ActorID;
+
+pub trait OperatorSet {
+    fn add_operator(&mut self, operator: ActorID);
+    fn remove_operator(&mut self, operator: &ActorID);
+    fn contains_actor(&self, operator: &ActorID) -> bool;
+}
+
+/// Maintains set-like invariants in-memory by maintaining sorted order of the underlying array
+/// Insertion and deletion are O(n) operations but we expect operator lists to be a relatively small size
+/// TODO: benchmark this against some other options such as
+/// - BTreeSet in memory, Vec serialized
+/// - BTreeSet in memory and serialization
+/// - HashSets...
+/// - Hamt<ActorID, ()>
+/// - Amt<ActorID>
```

Comment:
Please file an issue for this, but meanwhile I think the ordered list approach seems fine.

Reply:
https://github.com/helix-onchain/filecoin/issues/155